doc/help/kdb-record.md

```diff
@@ -0,0 +1,144 @@
+- `export  [--replay|--diff] [session_name]`: creates a file with the changes in the session.
+- `import [--disable-validation] <file_name>`: imports files created by the `export` command.
+
+## DESCRIPTION
+
+The `kdb record` utility is used to manage session recordings.
+Only a single recording session can be active per user at a time.
+A user can have multiple non-active sessions.
+
+If a session is active, every change to the keys, values and meta-data stored in Elektra will be logged into that session.
+
+## ENVIRONMENT VARIABLES
+
+- `ELEKTRA_RECORD_ACTIVE_SESSION`: If this environment variable is set, the specified session will be the active session.
+
+## COMMON PARAMETERS
+
+Most commands allow specification of the name of the session they should work against.
+If no session name is provided, the active session will be used.
+
+A notable exception is the `start` command.
+If no session name is provided, it will create a random name.
+
+## COMMANDS
+
+### `start [session_name]`
+The `start` command is used to start a new or resume an existing session.
+If the environment variable `ELEKTRA_RECORD_ACTIVE_SESSION` is set, this command has no immediate effect.
+As soon as the environment variable is unset, the session started or resumed with this command will be active.
+
+This command will accept a single optional argument that is the name of the session.
+If the name is not provided, a random name will be generated.
+
+On success, the name of the session will be printed to `stdout`, and the return code is `0`.
+
+### `stop`
+The `stop` command is used to stop the active recording session.
+The session can then be resumed using the `start` command with the appropriate session name.
+Stopping a recording session will not delete any recorded changes.
+
+### `reset [session_name]`
+The `reset` command will clear the specified recording.
+If no session name argument is provided, the currently active session will be reset.
+
+### `delete [session_name]`
+Deletes the specified recording session.
+It the optional argument is not provided, the current session will be deleted.
+If it is the currently active session, it will be stopped.
+
+### `mark [--meta meta_name] <key_name>`
+This command marks a key to be used for validation when importing the recording.
+By default, the values of a marked key will be compared when importing.
+The import will fail if the key has a different value.
+
+Using the `--meta` flag, one can specify to use the value of a specific meta instead of the value of the key.
+
+### `list`
+Lists all known recording sessions.
+
+### `name`
+Prints out the name of the currently active recording session.
+If no session is active, the return code is `1`, otherwise `0`.
+
+### `changes [session_name]`
+Shows all the changes in the specified session in human-readable format.
+Outputs to `stdout`.
+If no session is specified, the currently active session is used.
+
+### `export  [--replay|--diff] [session_name]`
```

Comment:
```suggestion
### `export  [--format=replay|diff] [session_name]`
```

Using separate `--replay`/`--diff` makes it seem like both can be used at the same time.

Reply:
Yes, good catch!